app/context_manager.py

```diff
@@ -0,0 +1,25 @@
+from os import remove, makedirs, path

```

Comment:
The import statement is incorrect. You should use 'from os import remove, makedirs' and 'import os.path as path' to correctly import the path module.